commands_to_load/Mod.py

```diff
@@ -79,5 +80,376 @@ def __init__(self, bot):
+            user = mentions[0]
+            args.remove(user.mention)
+
+        if args:
+            num = int(args[0])
+        else:
+            num = 10
+        logger.info('[Mod purge()] all arguments verified')
+        logger.info('[Mod purge()] to purge messages from: {}\nNumber of messages to purge: {}'.format(user, num))
+
+        # Check command will determine if message is from user. Will need counter
+        def check(m):
+            nonlocal num
+            if m.author == user and num > 0:
+                num -=1
+                return True
+            else:
+                return False
+            
+        # Call channel.purge() limit at 100 and bulk = True
+        deleted = await ctx.channel.purge(limit=100, check=check, bulk=True)
+        logger.info('[Mod purge()] purged messages: {}'.format(deleted))
+        
+        eObj = em(description='Purged {} messages from {}'.format(len(deleted), user), footer='This messages will self destruct in 5...')
+        await ctx.send(embed=eObj, delete_after=5.0)
+
+    @commands.command()
+    async def mute(self, ctx):
+        # Adds mute role to someone
+        
+        # Verify Minion
+        logger.info('[Mod mute()] mute function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod mute()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod mute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        # Check for mention
+        # If there get user
+        logger.info('[Mod mute()] checking for mention')
+        mentions = ctx.message.mentions
+        if len(mentions) != 1:
+            logger.info('[Mod mute()] no mention found. Informing user')
+            eObj = em(description='You need to @ mention the user to mute', footer='Invalid arguments')
+            await ctx.send(embed=eObj, delete_after=5.0)
+            return
+        else: 
+            user = mentions[0]
+        logger.info('[Mod mute()] user found through mention: {}'.format(user))
+
+        # Grab the Muted role
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+        logger.info('[Mod mute()] mute role found: {}'.format(MUTED_ROLE.id))
+
+        # Add muted role to user
+        await user.add_roles(MUTED_ROLE)
+        logger.info('[Mod mute()] adding muted role to user')
+
+        # Tell them in dm
+        logger.info('[Mod mute()] informing {} that they are muted in dm'.format(user))
+        eObj = em(title='Moderator action was taken against you', colour=0xff0000, description='You\'ve been muted in the CSSS server. Message a minion to learn why and how to be unmuted', footer='Moderator action')
+        await user.send(embed=eObj)
+        #TODO add something ^ to react to msg council for reconsideration slash reach out to u

```

Comment:
What's to be done? If I'm interpreting correctly, some kind of avenue of appeal?
Will take a 2nd look and hopefully approve this weekend.

Reply:
meant to remove this. its done now.